docs/versioned/admin/admin-overview.md

```diff
@@ -0,0 +1,130 @@
+
+Essentially, Knative aims to extend Kubernetes, and build on existing capabilities where feasible. It has two main underlying components that support plugging in multiple underlying transports within the same cluster:
+
+- Serving: Pods and pluggable network ingress routes.
+- Eventing: Pods and pluggable message transports (e.g. Kafka, RabbitMQ)
+
+Knative has default lightweight implementations if you don't already have a solution.
+
+## Installing Knative
+
+You install Knative using YAML files and other resources either aided or not by the Knative Operator. The Knative Operator is a custom controller that extends the Kubernetes API to install Knative components. It allows you to automate applying the content, along with patching the contents to customize them. You install the Knative Operator either by using the Knative CLI Operator Plugin or by using KS8 Manifests or by Yelm.
+
+Here are the considerations for installing using YAML or the Knative Operator:
+
+| YAML-based install | Knative Operator install|
+| --- | --- |
+| You can see exactly what you get. | You specify choices at a higher level. |
+| You can adjust any parameters by editing them directly. | Not every setting is exposed. |
+| If you make changes, you have to keep track of what you changed when you want to upgrade. | It's easy to separate your customizations from the base installation. |
+| Version and audit control as YAML files are stored in a GitHub repository.| No version or audit control. |
+
+You can install Knative in the following ways:
+
+- Use a [YAML-based installation](/install/yaml-install/README.md) with `kubectl`.
+
+    This option is the most useful if you're using delivery solutions such as Flux or ArgoCD to apply manifests checked into a Git repository. This is the lowest common denominator approach, giving you granular control of the process and resource definitions.
+
+- Install the [Knative Operator](/install/operator/knative-with-operators.md) using Manifests or Yelm, and then use `kubectl` to install Knative components.
+
+    This option alleviates complexity by using the Knative Operator, while still enabling purpose-built manageability using popular tools. It also gives you a separation of the core Knative application definition and the ConfigMap and other changes you make.
+
+- Install the [Knative Operator CLI plugin](/install/operator/knative-with-operator-cli.md) and install the Knative Operator and the Knative CLI `kn` to  Knative components.
+
+    This is the easiest install option and suitable for using if customization is not a concern.
+
+The following table summarizes the options.
+
+| Install option | Resources | kubectl CLI | kn CLI |
+| --- | --- | --- | --- |
+| YAML-based | All YAML prepared | Install components | not used |
+| Knative Operator | Install Knative Operator using Manifests or Yelm |Install components | not used |
+| Knative Operator | Install Knative Operator CLI plugin | not used | Install components |
```

Comment:
This table is kind of weird, particularly since the previous table was column-major, but this one is row-major.

Reply:
I'd probably drop this table, though the other table seemed informative.